projectorganizer/src/prjorg-project.c

```diff
@@ -310,7 +312,7 @@ static void regenerate_tags(PrjOrgRoot *root, gpointer user_data)
 }
 
 
-void prjorg_project_rescan(void)
+void prjorg_project_rescan(gchar **open_files)
```

Comment:
Rename to `session_files`

Reply:
In addition, I'd suggest converting it to a static function called `rescan_project` and having a separate `prjorg_project_rescan()` without any arguments and just calling `rescan_project(NULL);` inside it. This way you avoid the diffs adding the `NULL` parameter across the project.